node.gyp

```diff
@@ -720,6 +723,7 @@
           'libraries': [
             'Dbghelp',
             'Psapi',
+            'Winmm',
```

Comment:
If this is necessary to build v8_base, shouldn't this be placed in the v8 gypfiles? cc @targos @ryzokuken

Reply:
Possibly. What exactly in V8 uses Winmm APIs?

Reply:
https://github.com/nodejs/node/issues/28845#issuecomment-558438817 was the initial explanation as to why I had to add it at this level, so potentially there is something wrong in the gyp configuration of v8?

Regarding what is using it exactly, it's the use of `__imp_timeGetTime`. From the errors (as shown in https://github.com/nodejs/node/issues/28845#issue-472387811) I Googled it and the solution was to add Winmm.

Reply:
Is this blocking?